openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -177,6 +177,18 @@ def author_view(self, context=None):  # pylint: disable=unused-argument
         """
         Renders author view for Studio.
         """
+        return self.studio_view_fragment()
+
+    def preview_view(self, context=None):  # pylint: disable=unused-argument
+        """
+        Renders preview inside Studio.
```

Comment:
@ziafazal I think it would be worth mentioning that this is used when `DiscussionXBlock` is embedded by another XBlock (such as `GroupProjectXBlock`), and perhaps also explain what happens when this view is not present. That would make it immediately obvious why this block needs both an `author_view` and a `preview_view`.